Opentrons/opentrons
# feat(api): Add more pick up tip config elements to pipette config

These will need to change per-model in upcoming hardware revisions, so get ahead
of the 8 ball and put them in the configs rather than hardcoded.

Testing:
- Make sure we can pick up tips still and nothing crashes
- Change some numbers in the config and see that it changes
